ui/ui-components/pages/model/[modelId]/source/[sourceId]/multipleProperties.tsx

```diff
@@ -199,6 +197,12 @@ export default function Page(props) {
               onChange={(e) => setKey(e.target.value)}
               disabled={disabled}
             />
+            {!exactProperty && key !== column ? (
+              <Alert variant="info" className="mt-2">
+                Property with key "{column}" already exists, suggesting "{key}"
+                instead.
+              </Alert>
+            ) : null}
```

Comment:
I wonder if the tooltip component could be useful here https://react-bootstrap.netlify.app/forms/validation/#tooltips

Reply:
I could definitely move this bit of info into a form tooltip, but I'm not sure that fits. It looks like our choices for the tooltip are `valid | invalid` and I don't know if what we're communicating fits into that. We're saying essentially: "this value looks different because we needed to pick something that is available in the grouparoo app". Since it's just a small informational tidbit, I thought using the `info` variant would be the semantically consistent approach.

The tooltip/feedback component seems to be more focused on form validation and user input. I think it has the right look stylistically, and if it had the full `primary | info | warning | error | etc` options that alert does I think it would work. But as-is, I'm not sure how to map our key "suggestions" into the green/red options.

Reply:
I'll play with it and take some screenshots